app/models/user_group.rb

```diff
@@ -4,12 +4,12 @@ class UserGroup < ActiveRecord::Base
   include Linkable
   include PgSearch
 
-  has_many :memberships
+  has_many :memberships, dependent: :destroy
   has_many :active_memberships, -> { active.where(identity: false) },
            class_name: "Membership"
   has_many :users, through: :memberships
-  has_many :classifications
-  has_many :access_control_lists
+  has_many :classifications, dependent: :restrict_with_exception
```

Comment:
Yes. However, we're not using this column (we're stuffing it in metadata right now, since it's a list of user_group_ids rather than a single one). I've been thinking about converting that into a normal join table, shall I create an issue for that?